blocks/logic.js

```diff
@@ -231,7 +231,7 @@ Blockly.defineBlocksWithJsonArray([  // BEGIN JSON EXTRACT
         "name": "ELSE"
       }
     ],
-    "output": true,
+    "output": null,
```

Comment:
Yeah.  This passed the logic blocks generator tests (so I didn't notice it earlier), but threw in an error in the texts blocks generator tests.